kratos/python_scripts/eigen_solver_factory.py

```diff
@@ -19,6 +19,26 @@ def ConstructSolver(settings):
             return eigen_solver
         else:
             raise Exception("EigenSolversApplication not available")
+    elif solver_type == "eigen_feast" or solver_type == "eigen_feast_complex":
+        if kratos_utils.CheckIfApplicationsAvailable("EigenSolversApplication"):
+            import KratosMultiphysics.EigenSolversApplication as EiSA
+            if solver_type == "eigen_feast":
+                if settings.Has("symmetric") and settings["symmetric"].GetBool() and hasattr(EiSA, "FEASTSymmetricEigensystemSolver"):
```

Comment:
I think this check should be improved
Imagine someone changes the name of `FEASTSymmetricEigensystemSolver`. Then even if I set `symmetric` in my parameters I would get the other one!

Reply:
ok